utils/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
@vishesh92 is this boilerplate or does it serve any purpose? If you want an empty file you can touch the file and git add.

Reply:
We will need to add this file to module when we remove powermock. It specifies the MockMaker to use in that module for running tests.

more info here: https://github.com/apache/cloudstack/pull/7577/files#r1217685958